cairo/tests/ethereum/cancun/test_fork.py

```diff
@@ -868,6 +991,21 @@ def test_state_transition(self, cairo_run):
         assert chain.state == cairo_chain.state
         assert chain.chain_id == cairo_chain.chain_id
 
+    @pytest.mark.parametrize(
+        "zkpi_path",
+        list(Path("data/1/eels").glob("*.json")),
+        ids=[x.stem for x in Path("data/1/eels").glob("*.json")],
+    )
```

Comment:
You want to be able to run the state_transition state when you have downloaded and process a zkpi_file right ?
Why `data/1/eels` ?

Reply:
it's the default one actually if you do `zkpig generate`

Reply:
updating the README